core/src/main/java/io/kestra/core/tasks/flows/Flow.java

```diff
@@ -67,11 +67,17 @@ public class Flow extends Task implements RunnableTask<Flow.Output> {
     private Integer revision;
 
     @Schema(
-        title = "The input to pass to the triggered flow"
+        title = "The inputs to pass to the triggered flow"
     )
     @PluginProperty(dynamic = true)
     private Map<String, String> inputs;
 
+    @Schema(
+        title = "The labels to pass to the triggered flow"
+    )
+    @PluginProperty(dynamic = true)
+    private Map<String, String> labels;
```

Comment:
Flow can also have label so I wonder if this property would be better named `executionLabels` to avoid any mis-understanding.

Reply:
`labels` is best I think, we are already in the context of an execution